packages/expo/android/src/main/java/expo/modules/ExpoReactHostFactory.kt

```diff
@@ -106,9 +106,6 @@ object ExpoReactHostFactory {
           true,
           useDeveloperSupport
         )
-          .apply {
-            jsEngineResolutionAlgorithm = reactNativeHost.jsEngineResolutionAlgorithm
-          }
```

Comment:
This is no longer used https://github.com/facebook/react-native/pull/45269

Reply:
should we also change the code at https://github.com/expo/expo/blob/bc997e4f985fff0cbf0812f46df81b970f2cf8d7/packages/expo/android/src/main/java/expo/modules/ExpoReactHostFactory.kt#L63

Reply:
Not necessary because the value still exists inside `reactNativeHost`, from my understanding this was only removed from `ReactHost`

Reply:
oh yep that makes sense. thanks for confirming that.